ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -138,70 +144,73 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
       </ListPageHeader>
 
       {/*Race section*/}
-      <Section header='1. Race'>
+      <Section header='Race'>
         <div style={{display: 'flex', justifyContent: 'flex-start', flexWrap: 'wrap'}}>
           {AccountCreationOptions.race.map((race) => {
-            return this.createOptionCheckbox(race.label, 'race', race.value);
+            return this.createOptionCheckbox(race.label, 'race', race.value, race.value.toString());
           })}
         </div>
       </Section>
 
       {/*Ethnicity section*/}
-      <DropDownSection header='2. Ethnicity' options={AccountCreationOptions.ethnicity}
+      <DropDownSection header='Ethnicity' options={AccountCreationOptions.ethnicity}
                        value={demographicSurvey.ethnicity}
                        onChange={(e) => this.updateDemographicAttribute('ethnicity', e)}/>
-
-      {/*Gender section*/}
-      <Section header='3. Gender'>
-        <FlexRow style={{flexWrap: 'wrap'}}>
-          {AccountCreationOptions.gender.map((gender) => {
-            return this.createOptionCheckbox(gender.label, 'gender', gender.value);
-          })}
-        </FlexRow>
+      <Section header='Do you identify as lesbian, gay, bisexual, transgender, queer (LGBTQ),
+or another sexual and/or gender minority?'>
+        <div style={{paddingTop: '0.5rem'}}>
+          <RadioButton onChange={
+            (e) => this.updateDemographicAttribute('identifiesAsLgbtq', true)}
+                       checked={demographicSurvey.identifiesAsLgbtq}
+                       style={{marginRight: '0.5rem'}}/>
+          <label style={{paddingRight: '3rem', color: colors.primary}}>
+            Yes
+          </label>
+          <RadioButton onChange={(e) => this.updateDemographicAttribute('identifiesAsLgbtq', false)}
+                       checked={!demographicSurvey.identifiesAsLgbtq}
+                       style={{marginRight: '0.5rem'}}/>
+          <label style={{color: colors.primary}}>No</label>
+        </div>
+        <label></label>
+        <TextInputWithLabel labelText='If yes, please tell us about your LGBTQ+ identity'
```

Comment:
Given that this is how we're wording it, I agree with Greg's earlier comment that this could be much larger than 255 characters. I'd bump it all the way up to 4000 because storage is cheap and a lot of the time it will be empty anyways.